git/remote.py

```diff
@@ -58,6 +58,22 @@ def add_progress(kwargs, git, progress):
 
 #} END utilities
 
+def progress_object(progress):
```

Comment:
*Nitpick*: I would find `to_progress_instance` more descriptive when reading the code.